### PR TITLE
improvement(ui): remove project name from breadcrumbs

### PR DESCRIPTION
**Pull request by juliendonck**
closes issue #1326 
Replaces & closes stale PR #1468 

All credit to @finleygn, I just fixed the tests.

Signed-off-by: finleygn <finleygarton12@gmail.com>

### CODE REVIEW

**Comment by cloudhead** [comment]
It would be good if the project name was also clickable to go back to the root. I think the "Files" link is a bit obscure.

File: cypress/integration/project/source_browsing.spec.ts [hunk 1]
```diff
@@ -191,33 +191,6 @@ context("project source browsing", () => {
...
-              .pickWithContent(
-                ["project-screen", "file-view"],
-                "platinum / .i-too-am-hidden"
-              )
-              .should("exist");
-            commands.pick("project-screen", "file-view", "root-link").click();
-            commands
-              .pickWithContent(["project-screen", "file-view"], "README.md")
-              .should("exist");
-
```
Comment (rudolfs) [inline cypress/integration/project/source_browsing.spec.ts:220]: I'd rather we keep the test, but just click on the "Files" tab to get back to root:

```suggestion
// Going to a different path and then switching back to the root path
// shows the README again.
commands.pick("source-tree").within(() => {
  cy.contains(".i-am-well-hidden").click();
});
commands.pick("project-screen", "files-tab").click();
commands
  .pickWithContent(["project-screen", "file-view"], "README.md")
  .should("exist");

commands.pick("source-tree").within(() => {
  cy.contains(".i-too-am-hidden").click();
});
commands
  .pickWithContent(
    ["project-screen", "file-view"],
    ".i-too-am-hidden"
  )
  .should("exist");
commands.pick("project-screen", "files-tab").click();
commands
  .pickWithContent(["project-screen", "file-view"], "README.md")
  .should("exist");
```

**Comment by finleygn** [comment]
Oh whoops, never got round to fixing those tests. Thanks for picking up.

File: cypress/integration/project/source_browsing.spec.ts (hunk 1 above)
Comment (juliendonck) [reply cypress/integration/project/source_browsing.spec.ts:220]: :+1:

**Comment by juliendonck** [comment]
> Oh whoops, never got round to fixing those tests. Thanks for picking up.

@finleygn Totally understandable. I hope you don't mind me starting a new branch instead of branching off of yours. I'm a bit new to this, I'll add your sig-noff in the merge commit giving you credit where due

**Comment by juliendonck** [comment]
> It would be good if the project name was also clickable to go back to the root. I think the "Files" link is a bit obscure.

@cloudhead your mean this:
Screen Shot 2021-02-10 at 16 47 03

File: cypress/integration/project/source_browsing.spec.ts (hunk 1 above)
Comment (juliendonck) [reply cypress/integration/project/source_browsing.spec.ts]: https://github.com/radicle-dev/radicle-upstream/pull/1609/commits/dbcfebd6c09972c1aceb4a5f4a8e62ea674120ac

**Review by rudolfs — APPROVED** [review]

**Comment by cloudhead** [comment]
> > It would be good if the project name was also clickable to go back to the root. I think the "Files" link is a bit obscure.
> 
> @cloudhead your mean this:
> Screen Shot 2021-02-10 at 16 47 03

yeah

**Comment by juliendonck** [comment]
> > > It would be good if the project name was also clickable to go back to the root. I think the "Files" link is a bit obscure.
> > 
> > 
> > @cloudhead your mean this:
> > Screen Shot 2021-02-10 at 16 47 03
> 
> yeah



> > > It would be good if the project name was also clickable to go back to the root. I think the "Files" link is a bit obscure.
> > 
> > 
> > @cloudhead your mean this:
> > Screen Shot 2021-02-10 at 16 47 03
> 
> yeah

@cloudhead there's different ways to do this and some are hacky (and I can do) and others are more elaborate. I've created #1612 to track this and we can do this in a subsequent PR, so we can land this one.

**Review by MeBrei — APPROVED** [review]

**Comment by juliendonck** [comment]
closed by https://github.com/radicle-dev/radicle-upstream/commit/c8f89921901285494f79391812aef3e28acc8621

**Pull request closed without merging**